_core/smart/neon-node.md

```diff
@@ -27,64 +27,57 @@ git clone https://github.com/blockstack/stacks-blockchain.git
 cd stacks-blockchain
 ```
 
-Then build the project:
+### Configure your node to connect to the Neon network
 
-```bash
-cargo build
-```
+You'll need to update your node's configuration to connect to the public Neon testnet.
 
-Building the project on ARM:
-
-```bash
-cargo build --features "aarch64" --no-default-features
-```
+In the `stacks-blockchain` repository, open up the file `testnet/follower-config.toml`.
```

Comment:
looks like it's `follower-conf.toml`, not `config`

Reply:
Thanks - fixed